### ISSUE TITLE
Preview does not work

### ISSUE THREAD

**Issue by WaterChick** [AlonsoAliaga/AlonsoTags#16]
**Describe the bug**
I can't preview a tag even permission Preview: none

**Expected behavior**
I should be able to preview a tag

**Screenshots**
![image](https://github.com/AlonsoAliaga/AlonsoTags/assets/65216786/4d8dd6a3-183d-48ab-9330-3384ce9d99da)

**Important (please complete the following information):**
 - Java version: 17
 - Plugin version: 2.0.7-BETA-PRO
 - Server version: git-Purpur-1985 (MC: 1.19.4)*

**Comment by AlonsoAliaga** [comment]
Hello, whats your Spigot ID?
(Also, the message says you dont have permission to preview THAT tag, not the Preview permission)

**Comment by WaterChick** [comment]
Yeah i know, but i don’t see preview permissions anywhere on the documentation

**Comment by AlonsoAliaga** [comment]
Probably in the config.yml
Please, provide your spigot id

**Comment by WaterChick** [comment]
I am just working for the server, owner's ID: 671257

Can't find it in the config, idk if im blind

**Comment by AlonsoAliaga** [comment]
The buyer knows already when purchasing the resource that support is provided via Discord after verification 🤔
Well, check the config, one is the permission for Preview feature and other option is that the Tag permission is required too to preview a tag, check config.yml an option like that. For faster replies join us on Discord as is hard to ask questions here (I wont be on pc until night or tomorrow night)
